### ISSUE TITLE
Cannot quit / exit without killing the process

### ISSUE THREAD

**Issue by bitboxx** [passcod/cargo-watch#174]
Hi, so I am running `cargo watch -x test` to run unit tests for exercises I downloaded on [exercism.io](exercism.io). It works fine, but I can't quit using Ctrl-C. The only way I can stop it is killing the process eg. `pkill cargo-watch`.

Is this by design, or is this a bug?

```
› cargo-watch --version  
cargo-watch 7.8.0

› cargo --version 
cargo 1.52.0 (69767412a 2021-04-21)

› uname -a
Linux manjaro 5.10.34-1-MANJARO #1 SMP Sun May 2 11:29:56 UTC 2021 x86_64 GNU/Linux
```

**Comment by passcod** [comment]
That's a bug.

Can you try sending SIGINT with pkill?

Also can you show how you're running cargo watch, and a debug log

**Comment by bitboxx** [comment]
Hi @passcod,

Thanks for the reply, I appreciate it.

So I tested `› pkill --signal SIGINT cargo-watch` and it did work, so that lead me into thinking that the problem doesn't lie in cargo-watch. Then I closed and reopened Konsole. That solved the problem; Ctrl-C works again as expected. So it was a borked terminal.

I'm planning to use cargo-watch in the upcoming weeks, I'll keep an eye if the problem comes up again. And if it does, I'll do a further investigation to reproduce it.

For now, I'm closing this ticket. Will reopen if I have more actionable information.
